### PR TITLE
Removing all the jbake warnings 

### PR DESCRIPTION
**Pull request by jkremser**
plus enabling the include for remote adoc files

### CODE REVIEW

File: src/main/jbake/content/docs/user/user-guide.adoc [hunk 1]
```diff
@@ -10,14 +10,17 @@ Thomas Heute
 
 toc::[]
 
-include::metrics.adoc[]
+include::metrics.adocPart[]
 
-include::alerts.adoc[]
+// this now works
+//include::https://cdn.rawgit.com/hawkular/hawkular-metrics/master/api/metrics-api-jaxrs/README.adoc[]
```
Comment (jkremser) [inline src/main/jbake/content/docs/user/user-guide.adoc:16]: @theute If your idea was to include the remote `README.adoc` files, this is now possible (`allow-uri-read=true` in `jbake.properties` does the work)

**Comment by pilhuhn** [comment]
@Jiri-Kremser Can you rebase?

**Comment by jkremser** [comment]
> @Jiri-Kremser Can you rebase?

@pilhuhn rebased and added a commit that fixes all the new warnings

**Comment by pilhuhn** [comment]
Thanks @Jiri-Kremser

**Pull request merged**